x-pack/plugins/actions/server/plugin.ts

```diff
@@ -403,6 +404,9 @@ export class ActionsPlugin implements Plugin<PluginSetupContract, PluginStartCon
           );
         }
       },
+      isActionTypeEnabled: (id, options = { notifyUsage: false }) => {
```

Comment:
Change in tier triggers a Kibana restart every time right? So there shouldn't be a case where some actions are running in a stale environment and not hitting this?

Reply:
Nvm my comment! Addressed over Zoom.